classes/migration/upgrade/PKPv3_2_1SubeditorCategoriesMigration.inc.php

```diff
@@ -52,7 +51,7 @@ public function up()
     /**
      * Reverse the downgrades
      */
-    public function down()
+    public function down(): ?bool
     {
         throw new PKP\install\DowngradeNotSupportedException();
```

Comment:
Just noticed that needs a backslash (but is not actually related to the commit)

Reply:
Fixed, thanks!